x/storage/client/cli/tx.go

```diff
@@ -701,20 +717,32 @@ func CmdPutPolicy() *cobra.Command {
 
 func CmdDeletePolicy() *cobra.Command {
 	cmd := &cobra.Command{
-		Use:   "delete-policy",
+		Use:   "delete-policy [principal-type] [principle-value] [resource]",
```

Comment:
principal can be an address or a group id.

We can auto convert inside the cmd to reduce the number of parameters. 

For example:

```
./gnfd tx delete-policy 0xffffffffffffffffffffff
./gnfd tx delete-policy 3
```